### PR TITLE
Sorting on deeply nested resources

### PR DESCRIPTION
**Pull request by flmg**
Hi,

    /persons?include=pets,pets.toy&sort=-pets.toy.type

didn't work and produced the following query:
    
    select "person".* from "person" left outer join "pet" as "pets" on "person"."id" = "pets"."person_id" left outer join "toy" as "toy" on "pets"."id" = "toy"."pet_id" order by "pets"."toy"."type" desc

I used the `formatRelation` function so that the `order by` part is now correct:
    
    order by "toy"."type" desc

As usual, I added a test.

Cheers,

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -554,7 +554,10 @@ export default (Bookshelf, options = {}) => {
 
                 _forEach(sortValues, (sortBy) => {
 
-                    internals.model.orderBy(sortBy, sortDesc.indexOf(sortBy) === -1 ? 'asc' : 'desc');
+                    internals.model.orderBy(
+                        sortBy && internals.formatRelation(sortBy),
```
Comment (alechirsch) [inline src/index.js]: Not quite sure about this line, why not only pass in `internals.formatRelation(sortBy)`?

Comment (flmg) [reply src/index.js]: In the case where `sortBy` is an empty string or null/undefined, `formatRelation(sortBy)` returns `modelName.` which makes the query fail.

Comment (alechirsch) [reply src/index.js]: If that is the case, then I think we should put a conditional around the orderBy:
```
if (sortBy){
internals.model.orderBy(...)
}
```

**Comment by flmg** [comment]
@alechirsch Sorry for the delay, I was busy. I added the test around the sort value!

**Comment by alechirsch** [comment]
@jamesdixon Would you be able to look at a couple of the other PRs and consider publishing a release?

**Comment by jamesdixon** [comment]
@alechirsch unfortunately, I just don't have the time. I'd be more than willing to add you as an npm publisher if you give me your npm username. Sorry I can't be more help, but I have so many other priorities at the moment.

**Comment by alechirsch** [comment]
@jamesdixon, thanks for the offer. My npm user name is @alechirsch.

**Comment by jamesdixon** [comment]
@alechirsch you should be all set to publish. Thank you!

**Pull request merged**